commandServer/command_server.py

```diff
@@ -0,0 +1,79 @@
+import asyncio
+import configparser
+import logging
+from urllib.parse import urlparse, parse_qsl
+from routes import ROUTES
+import aiohttp
+from aiohttp import server
+from aiohttp.multidict import MultiDict
+
+
+class HttpRequestHandler(aiohttp.server.ServerHttpProtocol):
+    @asyncio.coroutine
+    def handle_request(self, request, payload=None):
+        response = aiohttp.Response(
+            self.writer, 200, http_version=request.version
+        )
+        response.add_header('Content-Type', 'application/json')
+
+        handler = ROUTES.get(request.path)
+        if not handler:
+            logging.error('Method not found for: %s' % request.path)
+            raise aiohttp.errors.HttpBadRequest('Bad Request')
+
+        method = getattr(handler(), request.method.lower(), None)
+        request_params = MultiDict(parse_qsl(urlparse(request.path).query))
+        method_response = method(request, request_params)
+        response.send_headers()
+        response.write(method_response)
+        yield from response.write_eof()
+
+
+class CommandServer(object):
+    _instance = None
+
+    def __init__(self, host=None, port=None, loop=None):
+        logging.info('Init Server on host %s:%s' % (host, port))
+        self._loop = loop or asyncio.get_event_loop()
+        self._server = self._loop.create_server(lambda: HttpRequestHandler(debug=True, keep_alive=75),
```

Comment:
why keepalive is 75?
Ideally we would like to have two HTTP servers: one for web sockets (with keepalive) and another for REST API